paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/MainActivity.kt

```diff
@@ -147,6 +158,11 @@ private fun MainScreen(items: List<MenuItem>) {
             items = groupedItems.getOrElse(MenuItem.Section.Internal) { emptyList() },
         )
 
+        Section(
```

Comment:
Nit: Should we move this above the `Internal` section? I’d like them to be somewhat separated.